flutter_local_notifications_windows/lib/src/details/notification_action.dart

```diff
@@ -0,0 +1,129 @@
+import "dart:io";
+
+import "package:xml/xml.dart";
+
+// NOTE: All enum values in this file have Windows RT-specific names.
+// If you change their Dart names, be sure to override [Enum.name].
+
+/// Decides how the [WindowsAction] will launch the app.
+///
+/// On desktop platforms, [foreground] and [background] are treated the same.
```

Comment:
Apologies if I've missed something but is due to limitation on supporting background isolates or something else? Wanted to check as the sentence looked odd as I thought this would only run on a desktop environment to begin with

Reply:
Taken from [here](https://learn.microsoft.com/en-us/windows/apps/design/shell/tiles-and-notifications/send-local-toast?tabs=desktop-msix#handling-background-activation): 

> For desktop applications, background activations are handled the same as foreground activations (your OnActivated event handler will be triggered). You can choose to not show any UI and close your app after handling activation.

Whereas for UWP apps: 
> When you specify background activation on your toast (or on a button inside the toast), your background task will be executed instead of activating your foreground app.

So this was me being overly descriptive. But, since Flutter officially does not UWP, I'll remove the `background` option and comment.